[confluence] fix misalignment of epg progressbar

tavoc opened the pull request:
The Progressbar in the EPG Grid View does not show the correct time. It seems to lag behind 10-15 minutes, because the Bar is left-aligned..

Changing it to center seems to work.

Paxxi commented:
Not My area of expertise, just wanted to say thank you for taking the time to make a PR :)

ronie commented:
you would also need to set  ```<left>``` to 0 else it's not properly centered.

xhaggi commented:
please squash it down to a single commit and and adjust your commit message to something proper.
`[confluence] fix misalignment of epg progressbar`

tavoc commented:
I hope thats right. If not  better close this and I do a new pull request i a single commit? First time git..., and the webUI didnt allow squash

xhaggi commented:
no it's wrong. please do the following:
```
git rebase -i HEAD~4
```

after that your editor should open, replace every "pick" in front of your commits with a "f" = fixup except the first line. Save your changes and now your commit history should only contains one commit.

Now we want to rename the commit message.
```
git commit --amend
```
change the commit message in your editor and save.

Now force-push your changes.
```
git push -f
```

That's all.

tavoc committed:
[confluence] fix misalignment of epg progressbar

The Progressbar in the EPG Grid View does not show the correct time. It
seems to lag behind 10-15 minutes, because the Bar is left-aligned..

Changing it to center seems to work.

tavoc commented:
thanks for the tip

xhaggi commented:
looks much better;)

xhaggi commented:
@ronie ping
